server/error-handler.js

```diff
@@ -0,0 +1,20 @@
+exports.postgresErrorHandler = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Bad request" });
+  } else if (err.code === "23502") {
```

Comment:
Why do you have two conditions if the outcome is the same? You may as well merge it.